Compile error with using TypeScript@3.7.2

Hey there!

I'm trying to update my extension to use TypeScript@3.7.2, and I'm getting a compile error from this package. I think the fix is to import as something other than `MappedPosition` position?

```
node_modules/vscode-chrome-debug-core/lib/src/sourceMaps/sourceMap.d.ts:1:10 - error TS2440: Import declaration conflicts with local declaration of 'MappedPosition'.

1 import { MappedPosition } from 'source-map';
           ~~~~~~~~~~~~~~


Found 1 error.
```

--- a/src/sourceMaps/sourceMap.ts
+++ b/src/sourceMaps/sourceMap.ts
@@ -2,7 +2,7 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
-import { SourceMapConsumer, MappedPosition } from 'source-map';
+import { SourceMapConsumer, MappedPosition as MappedPositionSM } from 'source-map';
 import * as path from 'path';
 
 import * as sourceMapUtils from './sourceMapUtils';
@@ -10,7 +10,7 @@ import * as utils from '../utils';
 import { logger } from 'vscode-debugadapter';
 import { IPathMapping } from '../debugAdapterInterfaces';
 
-export type MappedPosition = MappedPosition;
+export type MappedPosition = MappedPositionSM;
 
 /**
  * A pair of the original path in the sourcemap, and the full absolute path as inferred

